src/actions/Listing.js

```diff
@@ -4,15 +4,42 @@ import keyMirror from 'utils/keyMirror'
 
 import origin from '../services/origin'
 
+const featuredListingsGist = 'https://rawgit.com/sparrowDom/23f8b219567811221cba660039c7e438/raw/5610030ef20ba79245dbdb10af18d9d10c8fb98d/gistfile1.txt'
```

Comment:
Yes. And rather than a gist, why not point to the `hide_list` [branch](https://github.com/OriginProtocol/origin-dapp/tree/hide_list) as we did before? Then anyone can propose changes with PRs, and anyone with git priveledges can merge.